tests/FVTTests/homeView/testResizingOfDatasetTree.ts

```diff
@@ -0,0 +1,79 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2020
+ */
+import { expect } from 'chai';
+import { WebDriver, until, By } from "selenium-webdriver"
+
+import { loadPage } from 'explorer-fvt-utilities';
+import { getDriver, TEST_CONFIG } from '../testConfig';
+const { BASE_URL_WITH_PATH } = TEST_CONFIG;
+
+
+// Need to use unnamed function so we can specify the retries
+// eslint-disable-next-line
+describe('JES explorer page load', function () {
```

Comment:
Test needs appropriate name

Reply:
Updated name to `JES explorer sidebar actions.`